maven-build-scan/action.yml

```diff
@@ -0,0 +1,71 @@
+name: Publish Maven Build Scan
+description: Publish Maven Build Scan
+
+inputs:
+  build-workflow-filename:
+    description: 'Filename of the workflow where the maven-build-scan/save action was triggered'
+    required: true
+  develocity-url:
+    description: 'Develocity URL'
+    required: true
+  develocity-access-key:
+    description: 'Develocity access key'
+    required: false
+  tos-location:
+    description: 'Terms of Service location as an URL (https://foo.com/tos.html) or a Github repository file (/<owner>/<repo>/blob/<branch>/tos.html)'
+    required: true
+  develocity-allow-untrusted:
+    description: 'Develocity allow-untrusted flag'
+    default: 'false'
+  pr-comment-tos-acceptance-missing:
+    description: 'pull-request comment added when Terms of Service are not accepted ({0} in the value will be replaced by tos-location input)'
+    default: 'Please accept [Develocity Terms of Service]({0}) to get your pull-request Build Scan published by commenting this pull-request with the following message:'
+  pr-comment-tos-acceptance-request:
+    description: 'pull-request comment to accept the Terms of Service'
+    default: 'I have read Develocity Terms of Service and I hereby accept the Terms'
+  pr-comment-tos-acceptance-validation:
+    description: 'pull-request comment added when Terms of Service are accepted'
+    default: 'All Contributors have accepted Develocity Terms of Service.'
+  signature-branch:
```

Comment:
I wonder if "signature" is the best prefix here. This is the branch (and file) where we record TOS-acceptance. Maybe "tos-acceptance-file" and "tos-acceptance-file-branch" would be more self-explanatory as input names.